src/commands/database-import.ts

```diff
@@ -47,7 +47,7 @@ export const command = new Command("database:import <path> [infile]")
     "--filter <dataPath>",
     "import only data at this path in the JSON file (if omitted, import entire file)"
   )
-  .option("--chunk-size <mb>", "max chunk size in megabytes, default to 10 MB")
+  .option("--chunk-size <mb>", "max chunk size in megabytes, default to 1 MB")
   .option("--concurrency <val>", "concurrency limit, default to 5")
   .before(requirePermissions, ["firebasedatabase.instances.update"])
   .before(requireDatabaseInstance)
```

Comment:
In line 65, 
```
    const chunkMegabytes = options.chunkSize ? parseInt(options.chunkSize, 1) : MAX_CHUNK_SIZE_MB;
```

Reply:
Good catch, thanks!

Reply:
That is correct as is - the second arg of parseInt is the radix, not the default. Users will pass in the chunkSize arg in base 10, so it should stay as is

Reply:
That's the radix. Should be base 10.